metrics/src/main/java/com/expedia/adaptivealerting/metrics/functions/source/GraphiteQueryResult.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.metrics.functions.source;
+
+import lombok.Getter;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+import java.util.HashMap;
+import java.util.Iterator;
+
+public class GraphiteQueryResult {
+
+    /* graphite query result json format:
+     *  [ {"datapoints": [ [value1, timestamp1], [value2, timestamp2] ..],
+     *  "target": "time-series-key-identier", "tags" : "<tags from graphite function used in the render api>"} ]
+     */
+
+    private final String GRAPHITE_RESULT_DATAPOINTS_KEY = "datapoints";
+    private final String GRAPHITE_RESULT_TARGET_KEY = "target";
+    private final String GRAPHITE_RESULT_TAGS_KEY = "tags";
+
+    @Getter
+    private Datapoint Datapoint;
```

Comment:
The variables should follow camel case. should be datapoint here.

Reply:
Fixed